src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -142,9 +143,6 @@ func Run(updateHandler func(updated *Config)) error {
 
 func updateGlobals(cfg *Config) error {
 	globals.InstanceId = cfg.InstanceId
-	loc := &geolookup.City{}
-	loc.Country.IsoCode = cfg.Country
-	globals.SetLocation(loc)
```

Comment:
Actually I just stripped it -- the lat/long you could make more of an argument for caching, but less so the country I think.